kubeflow/tf-serving/prototypes/tf-serving-all-features.jsonnet

```diff
@@ -8,6 +8,16 @@
 // @optionalParam model_server_image string gcr.io/kubeflow-images-staging/tf-model-server:v20180227-master Container for TF model server
 // @optionalParam http_proxy_image string gcr.io/kubeflow/http-proxy:1.0 Container for http_proxy of TF model server
 // @optionalParam service_type string ClusterIP The service type for TFServing deployment.
+// @optionalParam s3_create_secret string true Whether or not to create a secret containing aws-access-key-id and aws-secret-access-key. Otherwise will use provided secret.
```

Comment:
Can you open up an issue to figure out a better long term solution? e.g. making it its own prototype?

Reply:
Opened #372